.github/RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,15 @@
+# Release Checklist
+
+Before releasing ensure that you have:
+- Run `.release/changelog.sh <PREVIOUS_RELEASE_TAG> <RELEASE_TAG>` to update CHANGELOG.md to contain all commits since the previous tag. Running without the previous release tag will get all commits.
+- Commit the updated changelog via PR with commit message "Preparing for release <RELEASE_TAG>"
+
+Releasing:
+- Go to: https://github.com/hyperledger/fabric-contract-api-go/releases
+- Select "Draft a new release"
+- Tag the version as vX.X.X. Note to include the character v. The repo uses a 3 number semantic versioning system.
+- Give the release a title of "Release <RELEASE_TAG>"
+- Writeup release notes:
+    - Include section called "Release Notes" containing the high-level view of changes made in the version that are of note.
+    - (Optional) Include a section called "Migration Notes" detailing "gotchas" for the user of migrating to the new version.
+    - (Optional) Include a section called "Bug Fixes" detailing important bug fixes in the version. Should be listed as bullet points with a link to the JIRA for that bug.
```

Comment:
are we planning on having a release notes .md file?

Reply:
I was planning on just using the secton of the github release process like node does